test/e2e/spworkflow/e2e_test.sh

```diff
@@ -26,7 +26,7 @@ function greenfield_chain() {
   # build Greenfield chain
   echo ${workspace}
   cd ${workspace}
-  git clone https://github.com/bnb-chain/greenfield.git
+  git clone https://github.com/alexgao001/greenfield.git
```

Comment:
Update to `bnb-chain` repo.

Reply:
yes, will updpate after there gnfd release